src/operator/ResourcesAccountType.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  ComponentColor,
+  DropdownMenuTheme,
+  MultiSelectDropdown,
+} from '@influxdata/clockface'
+import React, {FC, useContext} from 'react'
+import {OperatorContext} from './context/operator'
+import {AccountType} from 'src/types'
+
+const ResourcesAccountType: FC = () => {
+  const {accountTypes, setAccountTypes} = useContext(OperatorContext)
+  const availableTypes: AccountType[] = allAccountTypes
+
+  const handleSelect = (selectedOption: string): void =>
+    accountTypes.includes(selectedOption as AccountType)
+      ? setAccountTypes(
+          accountTypes.filter(x => x !== (selectedOption as AccountType))
+        )
+      : setAccountTypes([selectedOption as AccountType, ...accountTypes])
+
+  return (
+    <MultiSelectDropdown
+      style={{width: '220px', marginRight: '20px'}}
+      options={availableTypes}
+      selectedOptions={accountTypes}
+      onSelect={handleSelect}
+      menuTheme={DropdownMenuTheme.Onyx}
+      emptyText="Account Types"
+      buttonColor={ComponentColor.Primary}
+    />
+  )
+}
+
+export default ResourcesAccountType
+
+const allAccountTypes: AccountType[] = [
+  'cancelled',
+  'contract',
+  'free',
+  'pay_as_you_go',
+]
```

Comment:
might wanna move this up to the top prior to declaring the functional component. Not sure if it'll complain or not about trying to access a const prior to it being declared since variable hoisting is different for `var` and `const`

Reply:
Right on! I'll move it up!